packages/inferno-vnode-flags/src/index.ts

```diff
@@ -1,18 +1,22 @@
-export default {
-	Text: 1,
-	HtmlElement: 1 << 1,
+enum VNodeFlags {
```

Comment:
I think this needs to be const enum to make typescript compile bit operations values into integers